content/panorama/scripts/custom_game/hud.ts

```diff
@@ -110,9 +110,41 @@ function highlightUiElement(event: NetworkedData<HighlightElementEvent>) {
         if (duration) {
             $.Schedule(duration, () => removeHighlight({ path }));
         }
+
+        const isShopGuideItem = path.includes("HUDElements/shop/GuideFlyout/ItemsArea/ItemBuildContainer")
```

Comment:
I would just stick to `path.includes("HUDElements/shop/")` and embed it into the if statement.